### PR TITLE
TO BE CLOSED (SEE PR #13)

### PR DESCRIPTION
**Pull request by anayeaye**
Adds notebook that searches for HLS Sentinel-2 and Landsat data with CQL2 queries and dynamically mosaics search results.

### CODE REVIEW

**Comment by anayeaye** [comment]
Still trying to resolve `Make this Notebook Trusted to load map: File -> Trust Notebook`

File: hls-visualization.ipynb [hunk 1]
```diff
@@ -0,0 +1,720 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Endpoints\n",
+    "stac_base_url = \"https://staging-stac.delta-backend.com\"\n",
```
Comment (abarciauskas-bgse) [inline hls-visualization.ipynb]: minor but I was trying to be consistent in using `STAC_API_URL = ...` across notebooks

File: hls-visualization.ipynb [hunk 2]
```diff
@@ -0,0 +1,720 @@
...
+    "\n",
+    "ida_bbox = [-90.932637, 29.705366, -89.766437, 30.71627]\n",
+    "ida_temporal_range = [\"2021-07-01T00:00:00Z\", \"2021-10-28T00:00:00Z\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### First search the STAC api for items for specific dates available\n",
```
Comment (abarciauskas-bgse) [inline hls-visualization.ipynb]:
```suggestion
    "### First, search the STAC API for items for specific dates available\n",
```

File: hls-visualization.ipynb [hunk 3]
```diff
@@ -0,0 +1,720 @@
...
+    "This search body can be used with both the STAC and raster APIs, we will restrict the temporal range for mosaicing"
```
Comment (abarciauskas-bgse) [inline hls-visualization.ipynb]:
```suggestion
    "This search body can be used with both the STAC and raster APIs. We will restrict the temporal range for mosaicing."
```

Can we add a brief explanation for why we restrict the temporal range - is it just to limit the set of data included in the mosaic to something manageable?

**Comment by abarciauskas-bgse** [comment]
This works fine for me, after I added folium to the dependencies in requirements.txt, but I have seen that error before...

I made a few commits with the folium dependency and some minor text changes.

**Comment by anayeaye** [comment]
Changes were merged in #13

**Pull request closed without merging**